src/main/js/controls/Gantt/Track.js

```diff
@@ -86,28 +85,21 @@ const addTrackLabelEventHandler = (canvasSVG, labelObj) =>
     canvasSVG
         .selectAll(`.${styles.axisYTrackLabel} .tick text`)
         .each(function (displayVal) {
-            if (
-                !displayVal ||
-                !utils.isFunction(labelObj.onClick) ||
-                !hasMatchingTrackLabel(this, labelObj.display)
-            ) {
-                return;
-            }
-            if (
-                shouldTruncateLabel(
-                    labelObj.display,
-                    constants.DEFAULT_LABEL_CHARACTER_LIMIT
-                )
-            ) {
-                if (utils.isEmpty(d3.select(this).attr("aria-disabled"))) {
-                    d3.select(this).attr(
-                        "aria-disabled",
-                        !utils.isFunction(labelObj.onClick)
-                    );
-                }
-                d3.select(this).on("click", () => {
-                    labelObj.onClick(labelObj.display, d3.select(this));
-                });
+            if (!displayVal || !hasMatchingTrackLabel(this, labelObj.display)) {
+            } else {
+                shouldTruncateLabel(labelObj.display)
+                    ? d3
+                          .select(this)
+                          .style("cursor", "pointer")
+                          .on("click", () => {
+                              utils.isDefined(labelObj.onClick)
```

Comment:
Can you also add an inline comment here

// If the consumer provides onclick function, it will override default onClick functionality provided by Carbon-graphs

Reply:
[fixed](https://github.com/cerner/carbon-graphs/pull/242/commits/3d75d6b1109fb3da7bad04f51b1228bd6bcaaf93#diff-bdf6ac5b711158c299d4a3c2dce8913fR96)